### PR TITLE
To automate Region and Zone labels

### LINKED ISSUE #164: Automate Setting Node Labels for PowerVS
There are some well-known k8s labels that the RHACM UI uses ([failure-domain.beta.kubernetes.io/region](http://failure-domain.beta.kubernetes.io/region), [failure-domain.beta.kubernetes.io/zone](http://failure-domain.beta.kubernetes.io/zone), and [beta.kubernetes.io/instance-type](http://beta.kubernetes.io/instance-type)) and could be set for PowerVS nodes.

Ref: 
https://kubernetes.io/docs/reference/labels-annotations-taints/#topologykubernetesioregion

### PR DESCRIPTION
**Pull request by aishwaryabk**
Signed-off-by: Aishwarya Kamat <aishwarya.kamat@ibm.com>

### CODE REVIEW

**Comment by yussufsh** [comment]
Please create an issue to track the changes. Also I have a comment which should have been tracked in the issue.
`this feature should be optional. When details are not given we should just not label the nodes. This also achives backward compatibility`

**Comment by aishwaryabk** [comment]
> Please create an issue to track the changes. Also I have a comment which should have been tracked in the issue. `this feature should be optional. When details are not given we should just not label the nodes. This also achives backward compatibility`

I have created an issue on PowerVS - https://github.com/ocp-power-automation/ocp4-upi-powervs/issues/418
Should I also create one in the Playbooks repo?

**Comment by yussufsh** [comment]
It should be that way, powervs change will only include passing the region details. That is not true for powervm.
My main concern is as this playbook is used by other projects making it compulsory will not work when someone will update the hash there.

**Comment by aishwaryabk** [comment]
Created a new issue - https://github.com/ocp-power-automation/ocp4-playbooks/issues/164

**Comment by aishwaryabk** [comment]
Updated the changes.

File: playbooks/roles/ocp-customization/README.md [hunk 1]
```diff
@@ -16,6 +16,10 @@ Use this to modify kernel tunables for nodes/pods via [tuned operator](https://d
 Use this to deploy containerized version of [RSCT](https://www.ibm.com/support/knowledgecenter/SGVKBA) on every OCP node running on PowerVM infrastructure.  
 This is enabled by default when deploying OCP in PowerVM or PowerVS. Follow the instructions provided [here](/docs/rsct-deploy.md) to deploy RSCT on an existing OCP cluster. 
 
+**4. Automate setting of Node Labels
```
Comment (yussufsh) [inline playbooks/roles/ocp-customization/README.md]: Please correct the format.

File: examples/install_vars.yaml [hunk 2]
```diff
@@ -27,6 +27,8 @@ fips_compliant: false
 
 rsct_image: "quay.io/powercloud/rsct-ppc64le:latest"
 
+set_node_labels: {}
```
Comment (yussufsh) [inline examples/install_vars.yaml]: Can we name it simply 'node_labels'?

Comment (aishwaryabk) [reply examples/install_vars.yaml]: Sure. Implemented the change.

**Review by yussufsh — APPROVED** [review]
/lgtm
/cc @Prajyot-Parab @cs-zhang @sajauddin

**Review by Prajyot-Parab — APPROVED** [review]
/approve

**Pull request merged**